apps/vr-tests/tslint.json

```diff
@@ -1,11 +1,8 @@
 {
-  "extends": [
-    "office-ui-fabric-react-tslint"
-  ],
+  "extends": ["office-ui-fabric-react-tslint"],
   "rules": {
     "no-any": false,
-    "typedef": [
-      false
-    ]
+    "typedef": [false],
+    "jsx-no-lambda": false
```

Comment:
Do we need to turn of jsx-no-lambda here instead of using disable comments when appropriate?

Reply:
@kenotron